src/gui_interactive_controller.cpp

```diff
@@ -0,0 +1,171 @@
+#include "predefined.h"
+#include "gui_interactive_controller.h"
+
+#include "mainwindow.h"
+#include "./ui_mainwindow.h"
+
+
+#include"tobor_svg.h"
+
+
+
+void GuiInteractiveController::startGame() {
+	if (interactive_mode == InteractiveMode::NO_GAME) {
```

Comment:
@r0ost3r Please paste some code here for random numbers {0... n} with uniform distribution.
I guess we should use some time stamp as seed.